### PR TITLE
DOC-2847 Course update dates

### PR DESCRIPTION
**Pull request by catong**
DOC-2847. This PR updates doc for course updates to reflect changes made in TNL-4115. The date field now enforces a valid date. Edits include fixing the note position and numbering that was restarting after the nested unordered list.

REVIEWERS:
- [x] @efischer19 
- [x] @srpearce or @lamagnifica or @pdesjardins 

FYI @jaakana, @sstack22

### CODE REVIEW

**Comment by srpearce** [comment]
👍

File: en_us/shared/course_assets/handouts_updates.rst [hunk 1]
```diff
@@ -23,31 +23,44 @@ Add a Course Update
 **********************
 
 You add updates to notify learners of upcoming exams or deadlines, changes in
-the course schedule, or to make other announcements.
+the course schedule, or to make other announcements. Updates are visible to
+enrolled learners as soon as you post them.
+
+Each update requires a date for identification and sorting purposes. On the
+**Home** page of the course, updates are displayed in reverse chronological
```
Comment (lamagnifica) [inline en_us/shared/course_assets/handouts_updates.rst]: Sylvia's got me thinking about active voice...
maybe start with "In the LMS, updates display on the **Home** page in reverse chronological...

File: en_us/shared/course_assets/handouts_updates.rst [hunk 2]
```diff
@@ -23,31 +23,44 @@ Add a Course Update
...
+        quotation marks and apostrophes.
 
-  * This editor is like the :ref:`raw HTML editor<The Raw HTML Editor>` that is
-    provided for HTML components. To format your update, you must supply HTML
-    tags. The :ref:`visual editor<The Visual Editor>` is not provided.
+   * Specify a date for this update. By default, today's date is already
+     entered. You can change the date using the calendar tool, or by entering
+     some other valid date. This date is used only for identification and
+     sorting purposes, and does not control when the update becomes visible to
+     learners.
```
Comment (lamagnifica) [inline en_us/shared/course_assets/handouts_updates.rst:54]: I think that the "the update is visible to all enrolled learners as soon as you post it." statement would still be useful here.

**Comment by lamagnifica** [comment]
2 suggestions 👍

**Comment by efischer19** [comment]
This looks good to me, I don't think we need to call out the reverse-compatibility features (previously entered invalid dates are sorted as if the date is today but displayed with whatever invalid value they have, new values are forced to be valid).

👍

**Comment by catong** [comment]
@efischer19  Agreed re calling out the reverse-compatibility.  The behavior you put in place for those cases is logical and not disruptive, so I thought it would be unnecessarily confusing to refer to existing cases. It's also unlikely that course teams would refer to the documentation if they have already been posting course updates.

File: en_us/shared/course_assets/handouts_updates.rst (hunk 2 above)
Comment (catong) [inline en_us/shared/course_assets/handouts_updates.rst:54]: It feels too repetitive to say this 3 times in a short topic. I tried to put it back, but I decided that in this step it add more clarity to say that the specified date does not control visibility. In the next numbered step, we do say "Your update appears ... immediately." so I think all these mentions are sufficient in combination.

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1030

**Pull request merged**